proxy/src/main/java/io/openlineage/proxy/api/models/HttpLineageStream.java

```diff
@@ -0,0 +1,49 @@
+/*
+/* Copyright 2018-2022 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+package io.openlineage.proxy.api.models;
+
+import javax.ws.rs.WebApplicationException;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.client.ClientBuilder;
+import javax.ws.rs.client.Entity;
+import javax.ws.rs.client.Invocation;
+import javax.ws.rs.core.HttpHeaders;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import lombok.NonNull;
+import lombok.extern.slf4j.Slf4j;
+
+/** HttpLineageStream pushes events to http endpoint */
+@Slf4j
+public class HttpLineageStream extends LineageStream {
+
+  private final String url;
+  private final String apikey;
+
+  public HttpLineageStream(@NonNull final HttpConfig httpConfig) {
+    super(Type.HTTP);
+    this.url = httpConfig.getUrl();
+    this.apikey = httpConfig.getApikey();
+  }
+
+  @Override
+  public void collect(@NonNull String eventAsString) {
+    eventAsString = eventAsString.trim();
+    try {
+      ClientBuilder cb = ClientBuilder.newBuilder();
```

Comment:
I wouldn't construct a `client` on each request, but rather in the constructor `HttpLineageStream()` to reuse the reference.

Reply:
Nice recommendation for the optimal resource use